### PR TITLE
DEI leadership post

### PR DESCRIPTION
**Pull request by jnywong**
Blog post to acknowledge the gender gap in the current leadership team and our intentions to address it in the future.

### CODE REVIEW

**Comment by haroldcampbell** [comment]
Thank you for the draft @jnywong!
@choldgraf may I invite you to add/update?

**Comment by choldgraf** [comment]
This is a super important topic to me, and the post is an excellent start @jnywong - thank you for putting it together!

I agree w/ @jnywong's point that it's important for my voice to come through here as well. How about I take a pass at the post today (or if I run out of time, this weekend) to provide some historical context of how we've tried to improve diversity within the org, and also to provide an idea for how these issues align with our goals of growing our organizational maturity and resilience over the next 3 years.

If that sounds safe to try, I'll try to add my thoughts to the post ASAP, and invite @jnywong to give feedback/edits/etc to make sure they align with her ideas as well. Jenny would you be willing to be a co-author with me on this post?

**Comment by aprilmj** [comment]
The historical and future context would give this post even more meaning @choldgraf. Go for it!

I hope @jnywong is done working for the week, and I look forward to seeing this post evolve with y'all collaborating on it.

**Comment by jnywong** [comment]
Yes, @choldgraf (and anyone else), please feel free to directly contribute to this post in this PR 👍 

> Jenny would you be willing to be a co-author with me on this post?

Of course, great! Happy to provide feedback and edits after you have added your piece.

**Comment by choldgraf** [comment]
I've gone through the DEI post and made some edits with the following goals.

- Provide more historical context for how we've tried to build an inclusive culture and diverse team over the years.
- Add more detail for why we think diversity is important to 2i2c
- Note that improving the diversity of our staff and our leadership is an important part of the organizational growth of 2i2c in the coming years.
- Provide a few more concrete things we need to try to make initial progress.

I'd love to hear thoughts from @jnywong on what you think about these changes. Please make any edits or changes of your own to make sure you think we're delivering the right message effectively!

File: content/blog/2024/dei-leadership-strategy/index.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ We've also invested in creating an inclusive organizational culture via document
 
 ## The importance of diverse _leadership_
 
-However, over the past few months we have reflected on our organization's structure and future plans, and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
+However, over the past few months, we have reflected on our organization's structure and future plans and noticed a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
```
Comment (agoose77) [inline content/blog/2024/dei-leadership-strategy/index.md]: @choldgraf very minor point -- "noticed" implies it's suddenly occurred to us that there are very few women in the organisation. I don't feel confident that this accurately describes our position and how we got here. Rather, my assumption is that Jenny's original wording here better speaks to the fact that it's clearly an issue, but we have yet to solve it. This post, to my eyes, is saying "we need to solve it now". 

Please, _don't let me speak over more qualified voices on this subject_ (I have probably not been in any of these conversations), and _reject this if you think I've missed the point_, but I'll suggest a reword?

```suggestion
However, over the past few months, we have reflected on our organization's structure and future plans. In particular, we recognize that there is a worrying lack of women in staff leadership roles. This imbalance does not live up to our values of diversity in our team and service.
```

File: content/blog/2024/dei-leadership-strategy/index.md [hunk 2]
```diff
@@ -43,9 +43,9 @@ Here are some steps we aim to take to improve the diversity of our leadership ac
...
+4. **Continue improving our systems of delivery and work**. Finally, we believe that a crucial aspect of improving the diversity in our organization is to continue transforming our systems for delivery, reliability, and accountability across the organization. This effort is a crucial step for fostering an inclusive and equitable culture. We have [made a lot of progress in improving our delivery systems](../delivery-improvements/index.md) and will continue this improvement as an organization.
 
 We hope that this post both makes transparent a distinct lack of diversity within our organizational leadership and provides clarity for some of our plans to improve.
-We are excited about the organization that we have built, as well as its culture.
+We are excited about the organization that we have built so far, as well as being fiercely proud of our open-source culture and development practises.
```
Comment (agoose77) [inline content/blog/2024/dei-leadership-strategy/index.md:50]: > fiercely proud

:heart_eyes:

Comment (jnywong) [reply content/blog/2024/dei-leadership-strategy/index.md:50]: I'm pretty sure our whole team feels this way ❤️‍🔥

File: content/blog/2024/dei-leadership-strategy/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Below are a few ideas for how we aim to make improvements, and we invite feedback from others who are interested in helping us improve this aspect of our organization.
+
+## What we plan to do
+
+Over the next few months, we will work on a long-term DEI strategy in tandem with our sustainability strategy as we work [towards scalable and sustainable networks of community hubs](../funding-community-networks/index.md).
+
+Here are some steps we aim to take to improve the diversity of our leadership across the organization:
+
+1. **Define our goals for diversity**. First, we need a better understanding of the most important axes of diversity that we wish to design around. Gender is clearly a critical gap to cover, but there are many other important axes of diversity as well. For example, as an organization that serves a global community, it is important that we have global perspectives represented in our leadership as well.
```
Comment (agoose77) [inline content/blog/2024/dei-leadership-strategy/index.md]: Remove repeated "as well"
```suggestion
1. **Define our goals for diversity**. First, we need a better understanding of the most important axes of diversity that we wish to design around. Gender is clearly a critical gap to cover, but there are many other important axes of diversity as well. For example, as an organization that serves a global community, it is important that we have global perspectives represented in our leadership.
```

File: content/blog/2024/dei-leadership-strategy/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+2. **Define mechanisms for leadership and governance that includes representation along these axes**. Leadership at a staff level is one way to ensure representation of diverse perspectives, but there are many other ways to bring voices into the conversation as well. We aim to explore new ways of bringing diverse voices into the strategic direction of the organization and provide mechanisms for holding us accountable to our values. For example, we intend to grow a _board_ to guide 2i2c's strategy and hold its Executive Director accountable for achieving impact - this is another opportunity to bring diverse perspectives into our leadership.
```
Comment (agoose77) [inline content/blog/2024/dei-leadership-strategy/index.md]:
```suggestion
2. **Define mechanisms for leadership and governance that includes representation along these axes**. Leadership at a staff level is one way to ensure representation of diverse perspectives, but there are many other ways to bring voices into the conversation. We aim to explore new ways of bringing diverse voices into the strategic direction of the organization and provide mechanisms for holding us accountable to our values. For example, we intend to grow a _board_ to guide 2i2c's strategy and hold its Executive Director accountable for achieving impact - this is another opportunity to bring diverse perspectives into our leadership.
```

**Comment by jnywong** [comment]
Hey @choldgraf , really great to see you riff off my initial stub and add the much-needed backward- and forward-looking perspectives to round out this post!

I am pleased to see you explicitly committing to more concrete plans that we can hold ourselves to in future. 

There's rightly a lot of emphasis on bringing in new voices to the table, as we scale as an organisation, but we also need to remember to look within and retain brilliant talent over time as we mature https://github.com/2i2c-org/2i2c-org.github.io/pull/267/commits/cb7859e9ca8232dede1074e531e9e7440009cf95 .

For example, there have been previous efforts to map out career development for the engineering team and hopefully we can roll this process out across the whole team in the future. This is a "nice to think about, but not right now" item for this specific blog post.

Overall, I am happy with this going out once @damianavila gets a chance to take a look as requested!

File: content/blog/2024/dei-leadership-strategy/index.md (hunk 1 above)
Comment (choldgraf) [reply content/blog/2024/dei-leadership-strategy/index.md]: That works for me - this is something I've been stressed out about for a long time. I think that the slide deck sharpened attention on this point, as opposed to made it understood for the first time.

**Comment by choldgraf** [comment]
Sounds good, thanks @jnywong - also just noting we've had an issue about this for a few years now, that we could either repurpose or replace. This is another one of those "good intentions" issues that we haven't yet had time to flesh out because of the chaotic nature of small and growing organizations:

- https://github.com/2i2c-org/meta/issues/132

Rather than block this PR on @damianavila's approval, can we instead give him 24 hours to either approve or request changes? If he approves, no changes are requested, or changes are requested and we make them, we can post in ~24 hours.

Does that work?

**Comment by jnywong** [comment]
I have resolved all the comments.

Damián is happy with this going public with the caveat that we hold ourselves accountable for what we say we'll do in the medium term.

Leaving this to @choldgraf to merge in to go live 🚀

**Comment by choldgraf** [comment]
I looked over the post again, and don't have any worries about the stuff we're saying or committing to in there. I think it will likely take more time than we think it will (it always does), but O(year or two) feels safe to try and I think this comes through in the post.

Thanks folks for seeding this conversation, I'm really hopeful we can make progress at a strategic level in the coming weeks, and see the benefits of this in the coming months/years.

**Pull request merged**